### ISSUE TITLE
Steps provided to change the user's password do not work

### ISSUE THREAD

**Issue by MiguelCasaresRobles** [wazuh/wazuh-documentation#3241]
Hello team,

By following the documentation to change the user's password, it seems it does not work unless you specify the cluster name with the following option:

```
cd /usr/share/elasticsearch/plugins/opendistro_security/tools/
./securityadmin.sh -f <backup-directory>/internal_users.yml -t internalusers -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin.key -h <elasticsearch_ip> -cn cluster_1
```

This was not working for the Opendistro tab mentioned here: https://documentation.wazuh.com/4.0/user-manual/elasticsearch/elastic_tuning.html#change-elastic-pass

Regards,

Miguel Casares

**Comment by mimugmail** [comment]
It didn't work for me on a Debian 10 with following every step in manual multi-node deployment. 
You' get a stack trace without it:

```
root@siem-master:/usr/share/elasticsearch/plugins/opendistro_security/tools# /usr/share/elasticsearch/plugins/opendistro_security/tools/securityadmin.sh -f /root/internal_users.yml -t internalusers -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin.key -h 10.24.80.101
WARNING: JAVA_HOME not set, will use /usr/bin/java
Open Distro Security Admin v7
Will connect to 10.24.80.101:9300 ... done
ERR: Cannot connect to Elasticsearch. Please refer to elasticsearch logfile for more information
Trace:
NoNodeAvailableException[None of the configured nodes are available: [{#transport#-1}{fbUvsp4hRMK27o6DdfBR2g}{10.24.80.101}{10.24.80.101:9300}]]
        at org.elasticsearch.client.transport.TransportClientNodesService.ensureNodesAreAvailable(TransportClientNodesService.java:352)
        at org.elasticsearch.client.transport.TransportClientNodesService.execute(TransportClientNodesService.java:248)
        at org.elasticsearch.client.transport.TransportProxyClient.execute(TransportProxyClient.java:57)
        at org.elasticsearch.client.transport.TransportClient.doExecute(TransportClient.java:395)
        at org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:409)
        at org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:398)
        at com.amazon.opendistroforelasticsearch.security.tools.OpenDistroSecurityAdmin.execute(OpenDistroSecurityAdmin.java:524)
        at com.amazon.opendistroforelasticsearch.security.tools.OpenDistroSecurityAdmin.main(OpenDistroSecurityAdmin.java:157)
```

And this with `-cn`

```
root@siem-master:/usr/share/elasticsearch/plugins/opendistro_security/tools# ./securityadmin.sh -f /var/backups/internal_users.yml -t internalusers -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin.key -cn cluster-1 -h 10.24.80.101
WARNING: JAVA_HOME not set, will use /usr/bin/java
Open Distro Security Admin v7
Will connect to 10.24.80.101:9300 ... done
Connected as CN=admin,OU=Docu,O=Wazuh,L=California,C=US
Elasticsearch Version: 7.9.1
Open Distro Security Version: 1.11.0.0
Contacting elasticsearch cluster 'cluster-1' and wait for YELLOW clusterstate ...
Clustername: cluster-1
Clusterstate: GREEN
Number of nodes: 3
Number of data nodes: 3
.opendistro_security index already exists, so we do not need to create one.
Populate config from /usr/share/elasticsearch/plugins/opendistro_security/tools
Force type: internalusers
Will update '_doc/internalusers' with /var/backups/internal_users.yml
   SUCC: Configuration for 'internalusers' created or updated
Done with success
```

**Comment by davidfmiranda** [comment]
Hello @MiguelCasaresRobles and @mimugmail, as you report these commands will fail in distributed environments. The securityadmin script needs the cluster name to be indicated using the `-cn` option. Alternatively, the `-icl` option can be used to ignore the cluster name, this way, the command will run without indicating the cluster name.

I've already fixed this in #3244.

Regards,

David
